web3.js/src/message.ts

```diff
@@ -73,7 +80,15 @@ export class Message {
   constructor(args: MessageArgs) {
     this.header = args.header;
     this.accountKeys = args.accountKeys.map(account => new PublicKey(account));
-    this.recentBlockhash = args.recentBlockhash;
+    if (
+      Object.prototype.hasOwnProperty.call(args, 'latestBlockhash') &&
```

Comment:
Wherever this `hasOwnProperty` check appears, I think you can just write it as `args.hasOwnProperty(...)`, right?

Reply:
This one is an interesting one. If we use `args.hasOwnProperty(...)` TypeScript complains: 
```Do not access Object.prototype method 'hasOwnProperty' from target object.```

We could have disabled the rule that throws the error, but we weren't sure if that was really the best option. After doing some research on the error, we found an article that explained what the error was and how to address it -- which is what we used.

Reply:
Oh interesting. I've never seen this ESLint rule. https://eslint.org/docs/rules/no-prototype-builtins

I'm going to consider turning that off in our codebase.